### PR TITLE
Add postman + explore massa examples

### PR DESCRIPTION
**Pull request by aoudiamoncef**

### CODE REVIEW

File: technical-doc/api.rst [hunk 1]
```diff
@@ -15,34 +15,1170 @@ Massa JSON-RPC API is splitted in two parts :
...
 
-Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_  specification `here <https://raw.githubusercontent.com/massalabs/massa/main/docs/technical-doc/openrpc.json>`_.
+Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_  specification `here <https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
 
+Integrations
+============
+
+**JavaScript**: use `massa-web3.js <https://github.com/massalabs/massa-web3>`_.
+
+**Smart contracts**: use `massa-as-sdk <https://github.com/massalabs/massa-as-sdk>`_.
```
Comment (qdrn) [inline technical-doc/api.rst:25]: massa-as-sdk does not talk to the JSON-RPC-API, right ?

Comment (aoudiamoncef) [reply technical-doc/api.rst:25]: Nope, It's a section to redirect users to the right project, this why we choose the world `use`

Comment (qdrn) [reply technical-doc/api.rst]: Yes, but it does not really seem relevant to redirect users to `massa-as-sdk` as it's not related to the API.

Comment (aoudiamoncef) [reply technical-doc/api.rst]: We could remove it, no problem

Comment (qdrn) [reply technical-doc/api.rst]: I would remove it then.

**Pull request merged**